Detector/DetFCCeeBaseline1/compact/materials.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess this can be deleted and missing materials be added to `DetCommon/compact/materials.xml`?